paper-input-behavior.html

```diff
@@ -532,8 +532,10 @@
       this._ariaLabelledBy = labelledBy;
     },
 
-    _updateInputId: function() {
-      this._inputId =  'input-' + Polymer.PaperInputHelper.NextInputID++;
+    _generateInputId: function() {
+      if (!this._inputId || this._inputId === '') {
```

Comment:
@notwaldorf Maybe I'm missing the big picture ... ` if (!this._inputId)` wouldn't be enough ?

Reply:
i was being paranoid :D